RELEASE-NOTES.txt

```diff
@@ -6,6 +6,12 @@
 * Post Commenting: fixed issue that prevented selecting an @ mention suggestion.
 * Fixed an issue that could have caused the app to crash when accessing Site Pages.
 * Site Creation: faster site creation, removed intermediate steps. Just select what kind of site you'd like, enter the domain name and the site will be created.
+* Block editor: Added the Cover block
+* Block editor: Improve icon on the "Take a Video" media option
```

Comment:
This change I believe is Android only, since on iOS we are not showing icons on Media Options